python_modules/dagster/dagster/_core/scheduler/instigation.py

```diff
@@ -108,6 +108,9 @@ class SensorInstigatorData(
             # the last time the sensor was started
             ("last_sensor_start_timestamp", Optional[float]),
             ("sensor_type", Optional[SensorType]),
+            # the last time the tick completed evaluation, used to detect cases where ticks are
+            # interrupted part way through
+            ("last_tick_end_timestamp", Optional[float]),
```

Comment:
this is only set on success? should that be part of the name?

Reply:
Updated!